src/components/Characters/GenderFlower.js

```diff
@@ -1,35 +1,55 @@
-      />
-    </svg>
-  );
-}
-
-const GenderFlower = styled(GenderFlowerArtwork)`
+const GenderFlowerStyled = styled(animated.span)`
   position: absolute;
   z-index: ${zIndexArtworkMainDecor};
   color: ${props => props.theme.primaryColor};
+  transform-origin: '50% 50%';
 `;
 
+function GenderFlower(props) {
+  const { x } = useSpring({
+    from: {
+      x: 0
+    },
+    to: {
+      x: 1
+    }
+  });
+
+  const style = {
```

Comment:
Maybe a bit more meaningful name like _flowerAnimation_?

Reply:
After new implementation using *useTransition* hook, it's now just `props`.
```
//GenderFlower.js

<Wrapper key={key} style={props} className={className}>

```